Meadowlark-js/services/meadowlark-aws-lambda/serverless.yml

```diff
@@ -42,7 +42,7 @@ custom:
 
 provider:
   name: aws
-  runtime: nodejs16.x
+  runtime: nodejs14.x
   stage: ${opt:stage, 'local'}
   timeout: 10
```

Comment:
What do you think about trying to have multiple serverless config files? e.g. one with dynamo and one without.

Reply:
Quite likely. In the back of my mind, I've been thinking about how we might specify AWS Lambda but with different permutations of AWS services. Maybe similar configurations (like always using DocumentDB, but differing on using managed Kafka) go in the same Meadowlark service package with alternate serverless.yaml files, but less similar ones get their own package, for example a crazy direct-to-S3-for-storage one in something like "meadowlark-aws-lambda-s3-athena".